tests/unit/test_commands_validator.py

```diff
@@ -0,0 +1,46 @@
+import logging
+import unittest
+
+import pytest
+
+from autogpt.commands.validators import CommandsValidator
+from autogpt.logs import logger
+
+
+@pytest.mark.integration_test
+class TestCommandsValidator(unittest.TestCase):
```

Comment:
This is a mix of pytest and unittest style that we have been trying to move away from